src/js/common/components/RequiredLoginView.jsx

```diff
@@ -66,22 +66,15 @@ class RequiredLoginView extends React.Component {
           return <SystemDownView messageLine1="We couldn’t find your records with that information." messageLine2="Please call the Vets.gov Help Desk at 1-855-574-7286. We’re open Monday‒Friday, 8:00 a.m.‒8:00 p.m. (ET)."/>;
         }
 
-        // If va_profile has any other value, continue on to check if this user can
-        // use this specific service.
-        if (this.isServiceAvailable()) {
-          // If you have the required service, show the application view.
-          return this.props.children;
-        }
-
+        // Otherwise if va_profile is present and populated, show the application view.
         // If the required service is not available, the component will still be rendered,
-        // but we pass an `isDataAvailable` prop to child components indicating there is
-        // no data. (Only add this prop to React components (functions), and not ordinary
+        // but we pass an `isDataAvailable` prop to child components indicating whether 
+        // or not there is data.
+        // (Only add this prop to React components (functions), and not ordinary
         // DOM elements.)
+        const props = { isDataAvailable: this.isServiceAvailable() };
```

Comment:
I have the desire to rename a bunch of variables and prop names in this code, but maybe that's better done in a later PR that moves `AppContent` into shared code as @bshyong suggested:
`serviceRequired` and `requiredApp` should both just be `serviceName`
`isDataAVailable` should just be `isServiceAvailable`

Would you mind filing a tech debt ticket for that for the FE tech debt warriors to follow up on? (you can add labels `sitewide` and `front end`)